email-templates/ALL/DA/DOCUMENTATION_DECLINED.mjml

```diff
@@ -0,0 +1,128 @@
+<mjml>
+<!--SUBJECT: Din dokumentation er godkendt -->
+<!-- {{ company_name }} 			Name of seller company-->
+<!-- {{ invoice_number }} 	  Number of the invoice -->
+<!-- {{ payer_company_name }}     Name of the payer company-->
+<!-- {{ expiration_date }}       Date of expiration of the offer - current date + 1 day--> 
+<!-- {{ send_invoice }}       Blue button - link to draft invoice page in economic. Should say "Send faktura"--> 
+
+
+<mj-head>
+<mj-title>Din dokumentation er godkendt</mj-title>
```

Comment:
Filen hedder DECLINED, men der står godkendt?

Reply:
Err, maybe I did something wrong when copying this in. Will fix.